### ISSUE TITLE
bug: Unretriavable content on different-than-uploaded-from browser

### ISSUE THREAD

**Issue by tamas6** [fairDataSociety/fairdrive-theapp#480]
## 3 standalone probleme

1. when I was able to retrieve the files, even a 162mb big was proceeded, 
but **without a confirmation message, window,**  only later I realized it was indeed downloaded.

2.  at other desktop browser, (safari) I got log: `[Log] Error: Could not fetch directories & files! – Error: Incorrect chunk size (drive-5b01e44e94125811.js, line 1)
Error: Incorrect chunk size`
And the **POD is empty**

3. at mobile browser (safari) **POD is empty** and there is a UI mistake there, when you try to list pods, the listing "button" is not a button 
![notabutton](https://github.com/fairDataSociety/fairdrive-theapp/assets/81620050/482059bd-d808-4181-bfd9-5c74e6bed8a1)

the button function is unprecedented in fairdrive ecosystem for that styled TEXT, and the Create POD there with same typo and styling is also a mistake

**Comment by tamas6** [comment]
#468

**Comment by tamas6** [comment]
No.2 this  is the new log after 2weeks. <img width="760" alt="cannot  retrieve" src="https://github.com/fairDataSociety/fairdrive-theapp/assets/81620050/51b18ad6-39bc-4a3f-af09-0503ae3b1254">


 ## every bigger than 1MB file is lost in browser.
  2weeks ago I uploaded one 10mb file, and one 162mb. They are only CAN BE SEEN at FDA, but not able to copy or open it.

**Comment by plur9** [comment]
Thanks for the detailed report @tamas6 — triaging this now.

The three issues you've reported are actually distinct problems:

**Issue 2 — "Incorrect chunk size" (most critical):** This error surfaces when a file's stored chunks don't match the expected sizes during retrieval. Common causes:
- **Bee node version mismatch**: The Bee version used for upload vs retrieval has different chunk encoding behavior. Were the files uploaded on a different Bee node version than the one you're using to retrieve?
- **Postage batch expiry**: If the postage batch expired after upload, chunks may have been garbage-collected from some nodes, causing partial retrieval failures on certain browsers/devices (which reach different Swarm nodes).
- **Browser-specific chunk assembly**: Some browsers handle streaming differently, which can surface chunk boundary bugs that others mask.

**Issue 1 — No download confirmation**: This is a known UX gap — the file download completes silently with no toast/notification. Logged as a UX improvement.

**Issue 3 — Mobile Safari pod listing UI**: The text-styled pod listing element is not keyboard/touch accessible and visually ambiguous. Good catch — this is a separate accessibility/UI fix.

**Questions to help narrow down Issue 2:**
1. Is this still reproducible with the current Fairdrive at fairdrive.io? (There have been significant storage layer changes since this was filed.)
2. What Bee node version was running when you uploaded vs when you tried to retrieve?
3. Are you using your own node or a public gateway?

Linking to #508 which has related cross-device pod access symptoms.
